docs/src/main/tut/user-guide.md

```diff
@@ -441,7 +441,7 @@ For all `String`-based endpoints, Finch provides an `as[A]` method to perform ty
 is available for any `Endpoint[String]`, `Endpoint[Option[String]]` or `Endpoint[Seq[String]]` as
 long as a matching implicit `io.finch.DecodeEntity[A]` type-class is in the scope.
 
-This facility is designed to be intuitive, meaning that you do not have to provide a
+This facility is designed to be intuitive, meaning that you do not have to provide an
```

Comment:
I'm not sure about this one. It is correct because the next word does begin with a vowel, but in my mind the word that you pronounce next is `DecodeEntity` (which would make `a` the correct form). I'm kinda wondering with we actually want to drop the `io.finch` part, I'm not sure it really adds anything. Its not really a big deal and I'm fine with this change as is. Would be curious if anyone else had thoughts.